test/integration/02-user-wallet/02-bria.spec.ts

```diff
@@ -62,7 +64,8 @@ describe("BriaSubscriber", () => {
             receivedEvents.push(event)
             if (receivedEvents.length === nExpectedEvents) {
               setTimeout(() => {
-                resolver(receivedEvents)
+                resolve(receivedEvents)
+                clearTimeout(timeoutId)
```

Comment:
uhmm please use timeout promise, similar to lnd https://github.com/GaloyMoney/galoy/pull/2664/files#diff-f82eb815ee086a501049a55fd44d719dd309efdae7d7c812731e7bc5048ab766R1135 or waitFor https://chat.galoy.io/galoy/pl/ys1eq6uzwi8afeoe76j67e9dih